src/main/java/core/basesyntax/FindNumbers.java

```diff
@@ -11,6 +11,14 @@
      * числа 92, 18, 26 и 0.</p>
      */
     public int[] getAllNumbers(String text) {
-        return null;
+        String[] numbers = text.replaceAll("[^-\\d]", " ")
+                .replaceAll("\\s+", " ").trim().split(" ");
```

Comment:
```suggestion
        String[] numbers = text.replaceAll("[^-\\d]", " ").trim().split("\\s+");
```
try this way

Reply:
The solution is smart, thank you for advice!